### DISCUSSION TITLE
[Feature] Similar face suggestions

### DISCUSSION THREAD

**Discussion by mattheys** [immich-app/immich#13647] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Is it possible to add a feature that would suggest merging faces automatically and then you can approve or reject?

Under normal settings Immich likes to group my two boys as the same person so I've changed the settings to make the Maximum distance 0.4, however it would be handy if Immich could do what Google does and suggest faces it thinks might be the same person but have to manually accept them.  

Would this work easiest if on the Merge People page the faces were ordered by vector distance and not by what I assume is number of faces detected?

That way I don't need to wade through the hundreds of background people in the untagged faces to find duplicates of my children.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Dupe of  #4347/ #10534
